File: C++/Calificación.cpp
#include <iostream>
using namespace std;

int main()
{
    int ingreseCuantasCantidadDeNotas;
    float nota, promeioFinal;

    cout << "Cuantas notas vas a ingresar: ";
    cin >> ingreseCuantasCantidadDeNotas;

    for (int i = 1; i <= ingreseCuantasCantidadDeNotas; i++)
    {
        cout << "Calificación " << i << ": ";
        cin >> nota;
        promeioFinal += nota / ingreseCuantasCantidadDeNotas;
    }
    cout << "Promedio final " << promeioFinal << endl;
    if (promeioFinal < 3)
    {
        cout << "Tu calificación es F.";
    }
    else if (promeioFinal < 8)
    {
        cout << "Tu calificación es: D";
    }
    else if (promeioFinal < 11)
    {
        cout << "Tu calificación es: C";
    }
    else if (promeioFinal < 15)
    {
        cout << "Tu calificación es: B";
    }
    else
    {
        cout << "Tu calificación es: A";
    }
}

File: JavaScript/Calificación.js
let ingresaCantidadDenotas = parseInt(prompt("Ingresa cuantas notas vas ingresar: "))
let promedio = 0
let notas

for (let index = 1; index <= ingresaCantidadDenotas; index++) {
    let notas = parseFloat(prompt(`Calificación ${index}: `))
    promedio += notas / ingresaCantidadDenotas
}
document.write(`Tu calificación final es: ${promedio}`)